rs/sns/governance/unreleased_changelog.md

```diff
@@ -9,13 +9,40 @@ on the process that this file is part of, see
 
 ## Added
 
-Enable upgrading SNS-controlled canisters using chunked WASMs. This is implemented as an extension
+* Enable upgrading SNS-controlled canisters using chunked WASMs. This is implemented as an extension
 of the existing `UpgradeSnsControllerCanister` proposal type with new field `chunked_canister_wasm`.
 This field can be used for specifying an upgrade of an SNS-controlled *target* canister using
 a potentially large WASM module (over 2 MiB) uploaded to some *store* canister, which:
-* must be installed on the same subnet as target.
-* must have SNS Root as one of its controllers.
-* must have enough cycles for performing the upgrade.
+    * must be installed on the same subnet as target.
+    * must have SNS Root as one of its controllers.
+    * must have enough cycles for performing the upgrade.
+
+* Enable SNSs to opt in for
+[automatically advancing its target version](https://forum.dfinity.org/t/proposal-opt-in-mechanism-for-automatic-sns-target-version-advancement/39874)
+to the newest version blessed by the NNS. To do so, please submit a `ManageNervousSystemParameters` 
+proposal, e.g.:
+
+    ```bash
```

Comment:
We may want to make documentation somewhere else for this, and just link to it in our changelogs?

Reply:
I think adding this to the documentation is a good idea. I'd suggest that we keep the example DFX command here, still, so people can see it right in the proposal markdown, and hopefully feel inspired to submit such a proposal as soon as it's made available.

Reply:
Made a ticket to bring the SNS docs up to speed. https://dfinity.atlassian.net/browse/NNS1-3561